internal/webhooks/cluster.go

```diff
@@ -501,7 +494,7 @@ func (webhook *Cluster) validateTopologyVersion(ctx context.Context, fldPath *fi
 		return field.Invalid(
 			fldPath,
 			fldValue,
-			fmt.Sprintf("minor version update cannot happen at this time: %v", kerrors.NewAggregate(allErrs)),
+			fmt.Sprintf("version update cannot happen at this time: %v", kerrors.NewAggregate(allErrs)),
```

Comment:
I'm ok to defer to a follow-up PR, but the error messages seems not ideal:

"version update cannot happen at this time: blocking version update due to ControlPlane version check: ControlPlane version %q does not match the current version %q, blocking version update due to MachineDeployment version check:  there are MachineDeployments still completing a previous upgrade: [%s], there are MachinePools still completing a previous upgrade: [%s]"

Reply:
How would you like to optimize it?

@chrischdi Opinions?

Reply:
I'll give it a try

Reply:
Should be better now